Generate.py

```diff
@@ -199,7 +199,13 @@ def main(args=None) -> tuple[argparse.Namespace, int]:
                             elif category_name not in yaml:
                                 logging.warning(f"Meta: Category {category_name} is not present in {path}.")
                             else:
-                                yaml[category_name][key] = option
+                                if key == "triggers":
```

Comment:
I feel like this should be an `elif`

The nesting on this is already pretty bad :sweat_smile: